### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by MussieTeka**
This PR involves making changes to a SQL database schema and implementing queries based on the modified schema. 

## Schema Modifications

- Created tables "owners" and "species" with specific columns.
- Modified "animals" table: added foreign keys referencing other tables.

## Data Insertion

Data was inserted into the tables.

## Queries

Implemented queries involving JOIN operations and aggregations.

Please find these attached screenshots for query results.

queries1
queries2
queries3
queries4
queries5
queries6
queries7

### CODE REVIEW

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @MussieTeka , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a live demo section, Kindly remove it as you will not to deploy the application.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by MussieTeka** [comment]
Hello @binhussen,

Thank you for your feedback regarding the live demo section of my README file. I have taken note of your concern and understand that the project will not be deployed. However, I have already addressed this by stating in the README file that there is no live demo available. Here is the mentioned statement:

Live Demo

As feedback, it would be more helpful to raise an issue or comment instead of invalidating the submission.

For simplicity and to address your issue, I will go ahead and update the README file to remove the live demo section.

Best,
Mussie

File: schema.sql [hunk 1]
```diff
@@ -10,3 +10,20 @@ CREATE TABLE animals (
...
+  age INTEGER
+);
+-- Create the "species" table with the specified columns
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255)
+);
+-- Modify the "animals" table to set the "id" column as auto-incremented primary key
+ALTER TABLE animals
+ALTER COLUMN id SET DATA TYPE SERIAL PRIMARY KEY;
```
Comment (bobb-Rob) [inline schema.sql:29]: - [ ] You have not implemented the following requirements,  👇🏾 I see that you have done the first one below but the other three is left. Please implement them.

image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @MussieTeka, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (MussieTeka) [reply schema.sql]: Hello @bobb-Rob, thank you for your feedback. I have incorporated the missing commands.

**Review by amejid — APPROVED** [review]
Hi @MussieTeka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Correct workflow :heavy_check_mark: 
- SQL organized in multiple files :heavy_check_mark: 
- Queries returning correct results :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**